sub-logs/wacky.json

```diff
@@ -0,0 +1,17 @@
+{
+  "repo": "https://github.com/Arzejames/WackyDevWebsite",
+
+  "owner": {
+    "username": "arzejames",
+    "email": "turtles127855@gmail.com"
+  },
+
+  "target": {
+    "CNAME": {
+      "name": "wackydev", 
+      "value": "arzejames.github.io"
```

Comment:
Unable to view site as it's blocked by Google for phishing.

Reply:
Please provide screenshots.

Reply:
![image](https://github.com/is-a-good-dev/register/assets/117330195/62f0329b-4031-4a8d-88eb-12a456395567)